packages/office-ui-fabric-react/src/components/Icon/Icon.tsx

```diff
@@ -31,6 +45,15 @@ export const Icon = (props: IIconProps): JSX.Element => {
       className
     );
 
+    let iconFromCustomRendererFn: JSX.Element | undefined;
+    for (let i = 0; i < _iconRendererFns.length; i++) {
```

Comment:
for of instead of keeping track of i?

Reply:
Ahh, I had foreach but changed it because I was returning icon in the loop... which returned the icon from inner function rather than render.  Now that I'm wrapping icon in the same image div I can switch back to foreach.

Reply:
Foearch can't break early, but a for( of ) loop can